engine/Makefile

```diff
@@ -49,6 +50,7 @@ ifeq (${OS},)
   else
     OS         = UNIX
     FLAVOR     = $(shell uname)
+    ARCH       = $(shell uname -m)
```

Comment:
It's easy to forget you need to `shell` this, and it seems likely we'll utilize this same thing down the road to flag other behavior by cpu architecture, so I've added this as a variable on all Unix systems.